bootstrap.sh

```diff
@@ -94,11 +94,10 @@ SRCDIR=$PWD
 rm -rf $YUBIKIT_DIR
 rm -rf $YUBIKIT_OUT
 
-git clone -b master --single-branch https://github.com/YubicoLabs/yubikit-ios/ $YUBIKIT_DIR
+git clone -b 3.0.0-Preview --single-branch https://github.com/Yubico/yubikit-ios/ $YUBIKIT_DIR
```

Comment:
this is incorrect, it should be https://github.com/Yubico/yubikit-ios.git

Reply:
This works without `.git` as well. If its failing I can do a quick fix for this.

Reply:
it does not for me

Reply:
```
Cloning into 'Carthage/Checkouts/yubikit-ios'...
Warning: Permanently added 'github.com,140.82.112.3' (RSA) to the list of known hosts.
ERROR: Repository not found.
fatal: Could not read from remote repository.
```